test/suites/snapshots.sh

```diff
@@ -204,7 +208,7 @@ snap_restore() {
 
   ## prepare snap1
   lxc start bar
-  echo snap1 > state
+  echo snap1 >state
```

Comment:
@tugbataluy would you mind changing this line back to how it was, it doesn't seem related to the PR aim?